.gitignore

```diff
@@ -6,6 +6,7 @@
 *.log
 dist
 .idea
+mongo.js
```

Comment:
Yes I can change that and we don't want to upload it to git right? I mean it will be created when we run tests automatically. Also can we have the tests run as part of docker compose up.
@morrissinger  Also will the docker-compose up watch for code changes as of course we would not want to build every time we make a change locally while we are building stuff.

(Apologies in advance if that came as dumb question from this docker noob)